src/services/cronJobs/checkProjectVerificationStatus.ts

```diff
@@ -80,19 +84,18 @@ export const checkProjectVerificationStatus = async () => {
     maxDaysForSendingUpdateReminder,
   );
 
-  for (const project of projects) {
```

Comment:
This iteration was only running the first promise and the rest sometime in the future so latest test wasn't working. This could lead to false positives or unexecuted code.
Fixed with Promise.all